Support for cypress 9.x

I get an error when I want to use v1.6.0 with cypress v9.0.0:

```
 Could not resolve dependency:
npm ERR! peer cypress@"^2.1.0 || 3.x || 4.x || 5.x || 6.x || 7.x || 8.x" from cypress-localstorage-commands@1.6.0
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - chore(#382): Use Cypress v9 in E2E tests
 - chore(#382): Support any Cypress version greater than 2.1.0
+- chore(deps): Update devDependencies
 
 ## [1.6.0] - 2021-11-01
 ### Changed

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,6 +11,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Removed
 ### BREAKING CHANGES
 
+## [1.6.1] - 2021-11-11
+### Changed
+- chore(#382): Use Cypress v9 in E2E tests
+- chore(#382): Support any Cypress version greater than 2.1.0
+
 ## [1.6.0] - 2021-11-01
 ### Changed
 - chore(deps): Update devDependencies

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "cypress-localstorage-commands",
-  "version": "1.6.0",
+  "version": "1.6.1",
   "description": "Extends Cypress' cy commands with localStorage methods. Allows preserving localStorage between tests",
   "keywords": [
     "cypress",
@@ -43,7 +43,7 @@
     "prepare": "is-ci || husky install"
   },
   "peerDependencies": {
-    "cypress": "^2.1.0 || 3.x || 4.x || 5.x || 6.x || 7.x || 8.x"
+    "cypress": ">=2.1.0"
   },
   "devDependencies": {
     "@stryker-mutator/core": "5.4.1",

--- a/sonar-project.properties
+++ b/sonar-project.properties
@@ -1,6 +1,6 @@
 sonar.organization=javierbrea
 sonar.projectKey=javierbrea_cypress-localstorage-commands
-sonar.projectVersion=1.6.0
+sonar.projectVersion=1.6.1
 
 sonar.javascript.file.suffixes=.js
 sonar.sourceEncoding=UTF-8

--- a/test-e2e/react-app/package.json
+++ b/test-e2e/react-app/package.json
@@ -39,7 +39,7 @@
   "devDependencies": {
     "@cypress/webpack-preprocessor": "5.9.1",
     "babel-plugin-module-resolver": "4.1.0",
-    "cypress": "8.7.0",
+    "cypress": "9.0.0",
     "react-scripts": "4.0.3",
     "serve": "12.0.1",
     "start-server-and-test": "1.14.0"

--- a/test-e2e/typescript/package.json
+++ b/test-e2e/typescript/package.json
@@ -17,7 +17,7 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "5.2.0",
     "@typescript-eslint/parser": "5.2.0",
-    "cypress": "8.7.0",
+    "cypress": "9.0.0",
     "eslint": "7.32.0",
     "eslint-config-prettier": "8.3.0",
     "eslint-plugin-prettier": "3.4.1",

